woocommerce/woocommerce
# Add support for null inputs in wc_add_number_precision

### All Submissions:






### Changes proposed in this Pull Request:

https://github.com/woocommerce/woocommerce/pull/26965 added a type hint to the `wc_add_number_precision` function to prevent it from receiving empty strings. This pull request adds support for nulls too, returning 0 when one is passed to the function.

Additionally, unit tests are added for the modified functions.

Closes #33950.




### How to test the changes in this Pull Request:


1. Run `wp eval 'echo wc_add_number_precision(null);'`, it should display `0` instead of failing.
2. Run the unit tests in `tests/php/includes/wc-core-functions-test.php`

### Other information:




### FOR PR REVIEWER ONLY:
